### PR TITLE
Task#536167:[Refresh link]Replace bookmarks in repo

### PR DESCRIPTION
**Pull request by v-haiboz**
Task#536167:[Refresh link]Replace bookmarks in repo

### CODE REVIEW

File: aspnetcore/data/ef-rp/update-related-data.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ The page contains a hidden field (`<input type="hidden">`) for the course number
 
 ## Update the Course page models
 
-[AsNoTracking](/dotnet/api/microsoft.entityframeworkcore.entityframeworkqueryableextensions.asnotracking#Microsoft_EntityFrameworkCore_EntityFrameworkQueryableExtensions_AsNoTracking__1_System_Linq_IQueryable___0__) can improve performance when tracking isn't required.
+[AsNoTracking](/dotnet/api/microsoft.entityframeworkcore.entityframeworkqueryableextensions.asnotracking#) can improve performance when tracking isn't required.
```
Comment (Rick-Anderson) [inline aspnetcore/data/ef-rp/update-related-data.md:99]:
```suggestion
[AsNoTracking](/dotnet/api/microsoft.entityframeworkcore.entityframeworkqueryableextensions.asnotracking) can improve performance when tracking isn't required.
```

File: aspnetcore/razor-pages/filter.md [hunk 2]
```diff
@@ -174,7 +174,7 @@ The following code overrides the synchronous Razor Page filters:
 
 ## Implement a filter attribute
 
-The built-in attribute-based filter [OnResultExecutionAsync](/dotnet/api/microsoft.aspnetcore.mvc.filters.iasyncresultfilter.onresultexecutionasync?view=aspnetcore-2.0#Microsoft_AspNetCore_Mvc_Filters_IAsyncResultFilter_OnResultExecutionAsync_Microsoft_AspNetCore_Mvc_Filters_ResultExecutingContext_Microsoft_AspNetCore_Mvc_Filters_ResultExecutionDelegate_) filter can be subclassed. The following filter adds a header to the response:
+The built-in attribute-based filter [OnResultExecutionAsync](/dotnet/api/microsoft.aspnetcore.mvc.filters.iasyncresultfilter.onresultexecutionasync?view=aspnetcore-2.0#) filter can be subclassed. The following filter adds a header to the response:
```
Comment (Rick-Anderson) [inline aspnetcore/razor-pages/filter.md:177]:
```suggestion
The built-in attribute-based filter [OnResultExecutionAsync](/dotnet/api/microsoft.aspnetcore.mvc.filters.iasyncresultfilter.onresultexecutionasync?view=aspnetcore-2.0) filter can be subclassed. The following filter adds a header to the response:
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

File: aspnetcore/data/ef-rp/update-related-data.md (hunk 1 above)
Comment (v-haiboz) [reply aspnetcore/data/ef-rp/update-related-data.md]: Thanks for your reminding, it has been updated

File: aspnetcore/razor-pages/filter.md (hunk 2 above)
Comment (v-haiboz) [reply aspnetcore/razor-pages/filter.md]: Thanks for your reminding, it has been updated

**Comment by Rick-Anderson** [comment]
@serpent5  can you take `aspnetcore/host-and-deploy/windows-service.md` from #24727 and fix this merge conflict.

**Comment by serpent5** [comment]
> @serpent5 can you take `aspnetcore/host-and-deploy/windows-service.md` from #24727 and fix this merge conflict.

That should do it. 😅

**Comment by v-haiboz** [comment]
Redo

**Pull request closed without merging**